config/app_queue.php

```diff
@@ -62,5 +62,8 @@
 
 		// Additional plugins to include tasks from (if they are not already loaded anyway)
 		'plugins' => [],
+
+		// ignores task classes
+		'ignoredtasks' => [],
```

Comment:
```suggestion
		'ignoredTasks' => [],
```

Reply:
yeah I wasn't sure about which style to adopt since other options are a bit mixed :)